spec/system/search_pdc_results_page_spec.rb

```diff
@@ -37,6 +37,7 @@
     click_on "Year Published"
     expect(page).to have_content "View larger"
     expect(page).not_to have_content "[Missing]"
+    click_on "View larger"
```

Comment:
Unsure why the page is changed.  Maybe the ruby version change?  The data was still the same, but it would not pass on the main page